wom/src/main/scala/wom/package.scala

```diff
@@ -55,6 +55,7 @@ package object core {
   type EvaluatedRuntimeAttributes = Map[String, WomValue]
   // This one really does seem WOM specific
   type ExecutableInputMap = Map[String, Any]
+  type ResolvedImportRecord = String
```

Comment:
TOL: Type aliases are not type checked. This might be better as a value class - eg `final case class ResolvedImportRecord(importPath: String) extends AnyVal` (see "Correctness" in https://docs.scala-lang.org/overviews/core/value-classes.html#Correctness)